README.md

```diff
@@ -286,6 +286,32 @@ expect('hello world').toExclude('goodbye')
 expect('hello world').toNotContain('goodbye')
 ```
 
+### toIncludeKeys
+> `expect(object).toIncludeKeys(keys, [hasKey], [message])`<br>
```

Comment:
It's not a boolean switch, it's an optional comparator function like `toInclude` has 😊

I just updated the docs, reread and if you're still opposed I can take it out. I only added the parameter for some consistency with the `comparater` function in `toInclude`.